fusequery/query/src/interpreters/interpreter_factory.rs

```diff
@@ -31,4 +32,8 @@ impl InterpreterFactory {
             _ => bail!("Can't get the interpreter by plan:{}", plan.name())
         }
     }
+    pub fn get(ctx: FuseQueryContextRef, plan: PlanNode) -> Result<Arc<dyn IInterpreter>, ErrorCodes> {
+        InterpreterFactory::get_impl(ctx, plan)
+            .map_err(|exception| ErrorCodes::UnknownException(format!("{}", exception)))
```

Comment:
It would be better t use `anyhow` error like `.map_err(|exception| ErrorCodes::UnknownException(anyhow!("{}", exception)))` instead the `format!`?